rules/DowngradePhp70/Rector/FuncCall/DowngradeParenthesizedValueCallToCallUserFuncRector.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DowngradePhp70\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\PropertyFetch;
+use PhpParser\Node\Name;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @changelog https://stackoverflow.com/a/1596250/1348344
+ *
+ * @see \Rector\Tests\DowngradePhp70\Rector\FuncCall\DowngradeParenthesizedValueCallToCallUserFuncRector\DowngradeParenthesizedValueCallToCallUserFuncRectorTest
+ */
+final class DowngradeParenthesizedValueCallToCallUserFuncRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Downgrade calling a callable property disambiguated from calling a method using parentheses to call_user_func.',
+            [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+final class Foo
+{
+    /** @var callable handler for sending emails */
+    public $mailer;
+}
+
+$foo = new Foo;
+($foo->mailer)($args);
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+final class Foo
+{
+    /** @var callable handler for sending emails */
+    public $mailer;
+}
+
+$foo = new Foo;
+($foo->mailer)($args);
```

Comment:
```suggestion
```

Possibly left over

Reply:
Right, thanks. I fixed it in the tests but forgot about the example.